### ISSUE TITLE
Having a git issue that is preventing me from creating a pull request and not allowing me to merge with main. 

### ISSUE THREAD

**Issue by Thomas88s** [nss-day-cohort-46/How-to-Ask-for-Help#65]
**_Ticket MUST be completed before getting help from an instructor_**

Thomas WItherow

#### Group:
 
hairy-potter

#### Have you searched the other issue tickets? If not, do that first. 
- [x] Yes
- If No, stop and search.

####  REQUIRED Add Tags to Issue Ticket and Assign to Project Board
 

- [ x] Added label to specific project/assignment that you need help with
- [x ] Added to `c46 Help-Thread` project board

#### REQUIRED Describe your issue:
>>
Having an issue with my merge/pull request no showing up. Also merging with main doesn't seem to work.
terminal keeps telling me main is up to date but it is not.
#### REQUIRED List any error messages that appear:

➜  src git:(tw-hairyPotter) git checkout main
Switched to branch 'main'
Your branch is up-to-date with 'origin/main'.
➜  src git:(main) git pull origin main
From github.com:nss-day-cohort-46/hairy-potter-Thomas88s
 * branch            main       -> FETCH_HEAD
Already up-to-date.

#### REQUIRED What have you googled? **_(You must have at least 3 links related to your issue)_**

- [ ] 1. 
- [ ] 2. 
- [ ] 3. 

#### REQUIRED: What have you tried? Tell us everything you have tried. 
>>
Tried the whole push/pull cycle a few times
#### REQUIRED: Copy/paste link TO THE FILE on your branch that you are having issues with

**Your link:**

#### Code Snippet

```
PASTE YOUR CODE HERE BETWEEN THE BACKTICS
```

**Comment by jisie** [comment]
@Thomas88s What's the name of the branch you are trying to create a PR for?

**Comment by jisie** [comment]
I think you have a closed PR for `tw-hairyPotter`. So if you are using the same branch, you might need to reopen the that PR (and you can do that because the PR was closed without being merged into `main`).

**Comment by Scott47** [comment]
@Thomas88s is this issue resolved?

**Comment by Thomas88s** [comment]
no i just dig in deeper trying to fix it. I think ive been doing something because i have this problem across several repositories.

**Comment by Thomas88s** [comment]
The repository had to be re synced with my local main.  I reinitialized the branch by redoing the git init command and followed it through.
